common/help-on-spot-models/src/utils/getGeolocation.ts

```diff
@@ -2,29 +2,28 @@ import {AddressData} from "../models/RestModels";
 import {Config} from "aws-sdk";
 import Lambda from "aws-sdk/clients/lambda";
 
-require('dotenv').config();
-
 export async function getPointFromGeoservice(address: AddressData): Promise<number[]> {
+  // credentials are only needed for local setup
   const awsConfig = new Config({
     accessKeyId: process.env.AWS_ACCESS_KEY,
     secretAccessKey: process.env.AWS_SECRET_KEY,
     region: process.env.AWS_REGION
   })
   const lambda = new Lambda(awsConfig);
   const params = {
-    FunctionName: 'HoS-geolocation-dev',
+    FunctionName: process.env.GEOSERVICE_LAMBDA_NAME as string,
```

Comment:
Guess this needs to be set then in every post and patch lambda?

Reply:
I added it to every lambda, which uses gelocation service